pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -38,9 +38,10 @@ import (
 )
 
 const (
-	clusterManagerFinalizer   = "operator.open-cluster-management.io/cluster-manager-cleanup"
-	clusterManagerApplied     = "Applied"
-	clusterManagerProgressing = "Progressing"
+	clusterManagerFinalizer      = "operator.open-cluster-management.io/cluster-manager-cleanup"
+	clusterManagerApplied        = "Applied"
+	clusterManagerProgressing    = "Progressing"
+	workDriverConfigSecretSynced = "WorkDriverConfigSecretSynced"
```

Comment:
maybe just secret synced. We will need to also sync other secret for image pull credential later. These could be in the same condition, different reasons and messages.

FYI @zhiweiyin318

Reply:
updated this to a general value: "SecretSynced"